EIPS/eip-4362.md

```diff
@@ -0,0 +1,270 @@
+---
+eip: 4362
+title: EIP-4361 Capabilities Extension (CapGrok)
+description: A mechanism on top of Sign-In With Ethereum for informed consent to delegate capabilities with an extensible scope mechanism.
+author: Oliver Terbu (@awoie), Jacob Ward (@cobward), Charles Lehner (@clehner), Sam Gbafa (@skgbafa), Wayne Chang (@wyc)
+discussions-to: https://ethereum-magicians.org/t/eip-4362-siwe-capgrok/10627
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-20
+requires: 4361
+---
+
+## Abstract
+Sign-In with Ethereum (SIWE) describes how Ethereum accounts authenticate with off-chain services (relying parties) by signing a standard message format parameterized by scope, session details, and security mechanisms (e.g., a nonce). This specification describes a mechanism on top of SIWE to give informed consent to delegate capabilities with a certain extensible scope mechanism to an authorized delegee. The way how a delegee authenticates against the target resource is out of scope of this specification and depends on the implementation of the target resource. This specification is fully compatible with [EIP-4361](./eip-4361.md).
+
+## Motivation
+
+SIWE CapGroks unlock integration of protocols and/or APIs for developers by reducing user friction, onchain state and increasing security by introducing informed consent and deterministic capability objects on top of Sign-In With Ethereum (EIP-4361).
+
+## Specification
+
+This specification has three different audiences:
+- Web3 application developers that want to integrate CapGroks to authenticate with any protocols and APIs that support object capabilities.
+- Protocol or API developers that want to learn how to define their own CapGroks.
+- Wallet implementers that want to improve the UI for CapGroks.
+
+### Terms and Definitions
+
+- CapGrok - A SIWE Message complying with this specification, i.e., containing at least one CapGrok URI in the `Resources` section and the corresponding human-readable CapGrok Statement appended to the SIWE `statement`.
```

Comment:
I would propose the following breakdown for clarity:
* A CapGrok (or CapGrok Tuple if you want to be cheeky) expresses a capability consisting of a CapGrok URI + a CapGrok Statement (1:1 correspondence); optionally, it can be encapsulated in a CapGrok Details Object.
* A CapGrok Message is a SIWE message (string) that contains at least one CapGrok

I'm a little confused